### ISSUE TITLE
Remove dependency on `source-acceptance-test` in template `setup.py`

### ISSUE THREAD

**Issue by sherifnada** [airbytehq/airbyte#3679]
## Description
this package is not published so we should remove it until then.

Also, we should understand how this succeeded on the build. The generated scaffold template should have failed.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200368060087111/1200394569826999) by [Unito](https://www.unito.io)

**Comment by bazarnov** [comment]
I have a question, should `source-acceptance-test` be removed from all of the connector teampletes sources? 

- [ ] source-generic
- [ ] source-python
- [ ] source-python-http-api
- [ ] source-singer

**Comment by sherifnada** [comment]
@bazarnov after some consideration with the team, we've decided we should keep both of these for the following reason: 
1. acceptance tests can be used locally via pytest plugin which is convenient for local iteration
2. when running in CI, the acceptance test image is built before running the actual integration test for a connector
Since these two use cases are still valuable, we should keep them. apologies for the confusion here and thanks for looking into it!

Closing this issue
